test/acceptance/behave/components/ngsiv2/entities/update_append_attribute_by_id.feature

```diff
@@ -2869,7 +2893,7 @@ Feature: update or append an attribute by entity ID in NGSI v2. "POST" - /v2/ent
       | Fiware-Service     | test_update_op   |
       | Fiware-ServicePath | /test            |
       | Content-Type       | application/json |
-    When update or append an attribute by ID "room"
+    When update or append attributes by ID "room"
       | parameter            | value       |
       | attributes_name      | temperature |
       | attributes_value     | 80          |
```

Comment:
This feature, group all scenarios into a request, but in behave, the execution is very fast.